### ISSUE TITLE
/guzzlehttp/psr7/src/functions_include.php not found, is guzzle globally installed?

### ISSUE THREAD

**Issue by qbaze** [Elendev/nexus-composer-push#42]
guzzle/psr7 2.0.0 was released yesterday and all functions in the Guzzle\Psr7 namespace was removed with this release

https://github.com/guzzle/psr7/blob/2.0.0/CHANGELOG.md

**Comment by davidferlay** [comment]
+++ Just debuged that, happy to find fix commited few minutes ago:) awaiting release imppatiently @Elendev  :+1:

**Comment by Elendev** [comment]
Hello @davidferlay, I'll take a look as soon as you create a PR, and if everything is alright I'll gladly merge it!

**Comment by davidferlay** [comment]
Hello @Elendev , no wip on my side, I had (maybe wrongly) assumed that someone had already committed a fix (https://github.com/qbaze/nexus-composer-push/commits/master) and that only creating a new release was remaining to do

**Comment by qbaze** [comment]
@davidferlay @Elendev fix is there, needed to have it fixed asap as it's a dependency for us :)

**Comment by tm1000** [comment]
Seems the work on this has stalled. Issue is that this is broken in release 0.3 and in 0.4 and there's really no way around it without a proper fix

**Comment by thirsch** [comment]
As a workaround until a fix is released, you can simply add "guzzlehttp/psr7": "^1.8" in your dependencies to explicitly install the older, compatible version.

**Comment by josericardomcastro** [comment]
Same issue here... =/

**Comment by davidferlay** [comment]
Can you please share with us your vision @Elendev ?

**Comment by tm1000** [comment]
The solution from @thirsch is the best way to fix this until the PR is approved

**Comment by davidferlay** [comment]
yes that's for sure, I was inquiring about a more permanent/clean fix ^^

**Comment by jpruiz114** [comment]
Any updates on a solution for this issue? Thanks!

**Comment by jpruiz114** [comment]
Thanks for posting your solution @thirsch; it works fine. I hope a permanent solution is released soon.

**Comment by Elendev** [comment]
Hello @davidferlay, yep sorry I took time to respond.
I proposed a modification to the PR #43 to have more verbose logs in case of issues.
I'm not able to reproduce the issue, so I cannot test it, but I have confidence that what @qbaze did works well.

I'd like @qbaze to add the log and test it, just so if someone has an issue with this sensitive case, we'll have at least an insight on what the issue is.

When it's done I'll merge it and release a new version.
